src/games.py

```diff
@@ -248,13 +263,18 @@ def addGamesLinks(date = '', video_type = "archive"):
                     elif not future_video and not has_video:
                         add_link = False
 
+                    # if not live and st, et in game:
+                    #     st = calendar.timegm(time.strptime(game['st'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000
+                    #     et = calendar.timegm(time.strptime(game['et'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000
                     if add_link == True:
                         params = {
                             'video_id': game_id,
                             'video_type': video_type,
                             'seo_name': seo_name,
                             'has_away_feed': 1 if has_away_feed else 0,
                             'has_condensed_game': 1 if has_condensed_video else 0,
+                            'st' : calendar.timegm(time.strptime(game['st'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000,
+                            'et' : calendar.timegm(time.strptime(game['et'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000,
```

Comment:
You should probably be careful here and first test that game['et'] exists. For live games it probably isn't. I'd also check 'st' just to be on the safe side. I'd set them to None if they don't exist and in getGameUrl() add another check before I use them.